com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs

```diff
@@ -328,18 +339,61 @@ public void SetRenderer(int index)
             m_RendererIndex = index;
         }
 
+        /// <summary>
+        /// Returns the selected scene-layers affecting this camera.
+        /// </summary>
         public LayerMask volumeLayerMask
         {
             get => m_VolumeLayerMask;
             set => m_VolumeLayerMask = value;
         }
 
+        /// <summary>
+        /// Returns the transform that will act as a trigger for volume blending.
+        /// </summary>
         public Transform volumeTrigger
         {
             get => m_VolumeTrigger;
             set => m_VolumeTrigger = value;
         }
 
+        /// <summary>
+        /// Returns true if this camera requires to color information in a texture.
+        /// If enabled, color texture is available to be bound and read from shaders as _CameraOpaqueTexture after rendering skybox.
+        /// </summary>
+        public bool requiresVolumeFrameworkUpdate
+        {
+            get
+            {
+                if (m_VolumeFrameworkUpdateModeOption == VolumeFrameworkUpdateModeOverrideOption.UsePipelineSettings)
+                {
+                    return UniversalRenderPipeline.asset.volumeFrameworkUpdateMode == VolumeFrameworkUpdateMode.EveryFrame;
+                }
+                else
+                {
+                    return m_VolumeFrameworkUpdateModeOption == VolumeFrameworkUpdateModeOverrideOption.EveryFrame;
+                }
+            }
+            set => m_VolumeFrameworkUpdateModeOption = (value) ? VolumeFrameworkUpdateModeOverrideOption.EveryFrame : VolumeFrameworkUpdateModeOverrideOption.ViaScripting;
+        }
+
+        /// <summary>
+        /// Returns the current volume stack used by this camera
+        /// </summary>
+        VolumeStack m_VolumeStack = null;
+        public VolumeStack volumeStack
+        {
+            get => m_VolumeStack;
+            set => m_VolumeStack = value;
+        }
+
+        // This variable is used to detect when volume framework updates
+        // are switched from being enabled to disabled. In those cases
+        // we want to update the stack to capture the last state.
+        internal bool lastRequiresVolumeFrameworkUpdate { get; set; }
+
+        public List<Volume> volumeList { get; set; }
```

Comment:
I do not see any usages of this. It is also missing documentation?

Reply:
Good catch. That's something leftover from previous iteration.